doc/admin/conf_files/kdc_conf.rst

```diff
@@ -312,6 +313,11 @@ The following tags may be specified in a [realms] subsection:
     release 1.15 and later, it has the same meaning as
     **kdc_tcp_listen** if that relation is not defined.
 
+**kdc_tcp_listen_backlog**
```

Comment:
You are documenting this as a realm variable, when it is only read for [kdcdefaults].  Document it adjacent to kdc_max_dgram_reply_size in the kdcdefaults section.